lib/game_mode/regular.rb

```diff
@@ -0,0 +1,5 @@
+class Regular
+    def update(board:, position:, current_player:)
+        board.update(position: position, token: current_player.token)
+    end
+end
```

Comment:
If this is getting immediately consumed like this:  `Regular.new.update(...)`, you can shorten that by adding a class-level method:

```
class Regular
  # Same method signature
  #
  def self.update(board:, position:, current_player:)
    new.update(board: board, position: position, current_player: current_player)
  end
end
```

That way if you wind up needing to use internal state for this class later, you can still do that, but in the meantime you can access it with a slightly briefer public interface of `Regular.update`